generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            transforms.EnsureChannelFirstd(keys=["image", "label"]),
</pre><p>Please update the description above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>